tracer/test/Datadog.Trace.TestHelpers/SpanMetadataV0Rules.cs

```diff
@@ -179,6 +179,7 @@ public static Result IsCouchbaseV0(this MockSpan span) => Result.FromSpan(span)
                 .Matches(Name, "couchbase.query")
                 .Matches(Type, "db"))
             .Tags(s => s
+                .IsPresent("db.couchbase.seed.nodes")
```

Comment:
Should this be optional? I can see [here](https://github.com/DataDog/dd-trace-dotnet/pull/4278/files#diff-e9dd0d048fdc652c79ece31f9a3b16da757c44c47afa45ed5955ffc53e5dbabb) there is a case were it may be null